themes-default/slim/views/partials/showheader.mako

```diff
@@ -178,6 +178,7 @@
                                     <i>Preferred:</i> ${', '.join([capture(renderQualityPill, x) for x in sorted(preferred_qualities)])}
                                 % endif
                             % endif
+                                </td></tr>
```

Comment:
I hope this is okay, Vue was saying there was a missing ending tag for the `<tr>`.